File: app/src/main/java/ru/ls/donkitchen/rest/model/response/ReceiptDetailResult.java
package ru.ls.donkitchen.rest.model.response;

import com.google.gson.annotations.SerializedName;

/**
 * @author Lord (Kuleshov M.V.)
 * @since 13.10.15
 */
public class ReceiptDetailResult {

	/**
	 * id : 2
	 * categoryId : 1
	 * name : Уха по-старочеркасски
	 * imageLink : http://dk.lord-services.ru/uploads/e63cee0eff2abaa90ac36ce93defae1a.png
	 * ingredients :
	 * receipt : Мелкая рыба
	 * categoryName : Первые блюда
	 */

	@SerializedName("id")
	public int id;
	@SerializedName("categoryId")
	public int categoryId;
	@SerializedName("name")
	public String name;
	@SerializedName("imageLink")
	public String imageLink;
	@SerializedName("ingredients")
	public String ingredients;
	@SerializedName("receipt")
	public String receipt;
	@SerializedName("categoryName")
	public String categoryName;
	@SerializedName("views")
	public int views;
	@SerializedName("rating")
	public int rating;
}

File: data/src/main/kotlin/ru/ls/donkitchen/data/rest/response/ReceiptDetailResult.kt
package ru.ls.donkitchen.data.rest.response

import com.google.gson.annotations.SerializedName

/**
 *
 *
 * @author Lord (Kuleshov M.V.)
 * @since 16.10.16
 */
class ReceiptDetailResult {
    /**
     * id : 2
     * categoryId : 1
     * name : Уха по-старочеркасски
     * imageLink : http://dk.lord-services.ru/uploads/e63cee0eff2abaa90ac36ce93defae1a.png
     * ingredients :
     * receipt : Мелкая рыба
     * categoryName : Первые блюда
     */

    @SerializedName("id")
    var id: Int = 0
    @SerializedName("categoryId")
    var categoryId: Int = 0
    @SerializedName("name")
    var name: String = ""
    @SerializedName("imageLink")
    var imageLink: String? = null
    @SerializedName("ingredients")
    var ingredients: String? = null
    @SerializedName("receipt")
    var receipt: String = ""
    @SerializedName("categoryName")
    var categoryName: String? = null
    @SerializedName("views")
    var views: Int = 0
    @SerializedName("rating")
    var rating: Int = 0
}